xero_accounting.yaml

```diff
@@ -19841,6 +19956,23 @@ paths:
                             }
                           ]
                         }'
+  '/Reports':
```

Comment:
So `'/Reports':` sans params is an existing endpoint (which I just tested and realized isn't even working) that is supposed to hit a function called `getReportBASorGSTList` based on the report ID passed..

Considering BAS is only for AUS and GST is only for NZ I'm betting that is why they share the same `GET https://api.xero.com/api.xro/2.0/Reports/<report-id>` pattern and name.

It is one of two Report endpoints that don't follow the RESTful route of the named report.
In short of looking in the URL idk how you are supposed to know that GUID
Example:
```
GET https://api.xero.com/api.xro/2.0/Reports/TenNinetyNine
GET https://api.xero.com/api.xro/2.0/Reports/AgedPayablesByContact
GET https://api.xero.com/api.xro/2.0/Reports/BalanceSheet
```
And the two AUS / NZ tax report routes..
```
BAS Report- GET https://api.xero.com/api.xro/2.0/Reports/3d0a1240-e606-4fae-a823-77bcf79d5e79
GST Report - GET https://api.xero.com/api.xro/2.0/Reports/2be4a28b-467a-4bd1-baf8-8d6622a5b930
```
@kchandola - do you know if these UUID's are hardcoded? or Dynamic across every Xero instance? 🤔 

---
I'll test some changes and add to this PR because line:19959 breaks SDK generation as its a duplicate route def. I will fix the existing route, and add to this PR - glad you brought this one up :)

Reply:
@SerKnight not all reports need a GUID to run. Org level reports can run using org info.
- BAS report (AU) and GST (NZ) are the ones that need a GUID for a particular report, using a parameter called ReportID. 
- TenNinetyNine report doesn't require a GUID to run, we just need to know report year (eg 2021) and it should work for that org.
- BalanceSheet - No UUID required, although users can specify date or time period to get BS for the org.
- AgedPayablesByContact - Need a ContactID which is UUID. Not sure how that fits here.

Reply:
Hey @kchandola :) Sorta answered it. Main question is - is the UUID for BAS or GST static across all Xero instances.

Other then looking at that report in the UI, and plucking off the Report UUID to pass to the `/Reports/<uuid` API call how should developers find that?

Reply:
@SerKnight gotcha. So the UUIDs are specific to each report. Other than taking it from the URL on the UI, users can also hit the API endpoint to get a list of all reports` /Reports`, including their reportIDs.

Reply:
Ah thank you for clarifying - I will make sure the `GET /Reports` is working as expected and get all this merged asap!